sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageBuilder.java

```diff
@@ -85,7 +88,12 @@ public AzureBlobStorageBuilder pipeline(HttpPipeline pipeline) {
      */
     public AzureBlobStorageImpl build() {
         if (pipeline == null) {
-            this.pipeline = RestProxy.createDefaultPipeline();
+            this.pipeline = new HttpPipelineBuilder()
```

Comment:
This class and all others with a similar name are codegen, you'll need to work with @jianghaolu to update this appropriately.

For example Storage doesn't use the `RetryPolicy` in Azure Core, instead the `RequestRetryPolicy` is used.

Reply:
I don't think this code path should be reach when building one of the public clients, but want to make sure we build the right thing if we do.

Reply:
@alzimmermsft This code is copy / paste from the old `RestProxy.createDefaultPipeline()` code path that was called (which I have removed, hence the copy / paste). This means that there should be no change in behaviour before / after this code change. If the RetryPolicy should not be used, then that needs to be cleaned up consistently (but probably in a follow-up PR).

Reply:
This is an easy fix in the generator.